libs/wingsdk/src/std/number.ts

```diff
@@ -25,7 +25,7 @@ export class Number {
   /**
    * Parse a number from string.
    *
-   * @macro ((args) => { if (isNaN(args)) {throw new Error("unable to parse \"" + args + "\" as a number")}; return parseInt(args) })($args$)
+   * @macro ((args) => { if (isNaN(args)) {throw new Error("unable to parse \"" + args + "\" as a number")}; return new Number(args) })($args$)
```

Comment:
My bad, JS hates me. This fails under strict equality because this is actually creating a different object than a number. I hate that there are 9000 ways to do this conversion. Anyways, this *should* work.

```suggestion
   * @macro ((args) => { if (isNaN(args)) {throw new Error("unable to parse \"" + args + "\" as a number")}; return Number(args) })($args$)
```

Reply:
Oh totally forgot to go back into this and review if it failed or passed. Thanks for suggesting fix!